godot-codegen/src/generator/enums.rs

```diff
@@ -483,6 +519,23 @@ fn make_enum_doc(enum_: &Enum) -> Vec<String> {
 
     docs
 }
+/// Returns the hint string for the given enum.
+///
+/// Separate with commas, and remove the `<ENUM_NAME>_` prefix (if possible). 
+/// e.g.: "Left,Center,Right,Fill"
+fn make_enum_hint_string(enum_: &Enum) -> String {
+    let upper_cast_enum_name = enum_.godot_name.to_shouty_snake_case() + "_";
+    enum_.enumerators
+         .iter()
+         .map(|enumerator| {
+             enumerator.godot_name
+                       .clone()
+                       .trim_start_matches(upper_cast_enum_name.as_str())
+                       .to_pascal_case()
+         })
+         .collect::<Vec<String>>()
+         .join(",")
+}
```

Comment:
As mentioned, this logic should already exist in some form.

Reply:
I see. Can you reuse one of these though? 

https://github.com/godot-rust/gdext/blob/0b6b792d24018d4761d0412609e6cc4dd69c1a21/godot-codegen/src/conv/name_conversions.rs#L57-L105

If not, it should at least become a dedicated conversion function in that file (ideally with a comment on how it differs from the others). Thanks! 🙂